vue-frontend/src/components/home-new/LandingSection.vue

```diff
@@ -1,164 +1,177 @@
 <template>
-  <section>
+  <div class="tw-relative tw-px-0 lg:tw-py-[50px] tw-overflow-hidden">
+    <img
+      :src="bg2400"
+      :srcset="`${bg400} 400w, ${bg800} 800w, ${bg1200} 1200w, ${bg1200} 2400w`"
```

Comment:
`${bg1200} 2400w `?

Reply:
I was checking that 1200 width image is set on 2400 width screen

Reply:
It should 2400. How 1200 can fit in 2400? It will definitely cut from borders

Reply:
But 2400 width image does not fit on 2400 width screen